tests/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright (c) QuantCo 2023-2024
```

Comment:
This seems dubious. This used to work without this before - can you explained what changed?